src/table.py

```diff
@@ -2543,6 +2639,25 @@ def find_tables(
         "text_x_tolerance": text_x_tolerance,
         "text_y_tolerance": text_y_tolerance,
     }
+
+    old_quad_corrections = pymupdf.TOOLS.unset_quad_corrections()
```

Comment:
We should do:
```
old_quad_corrections = pymupdf.TOOLS.unset_quad_corrections()
try:
    ...
finally:
    pymupdf.TOOLS.unset_quad_corrections(old_quad_corrections)
```
Otherwise global state gets modified silently if an exception is thrown.